src/dimensions.js

```diff
@@ -19,6 +19,11 @@ jQuery.each( { Height: "height", Width: "width" }, function( name, type ) {
 
 				if ( jQuery.isWindow( elem ) ) {
 
+					// $( window ).outerWidth/Height return w/h including scrollbars (gh-1729)
+					if ( funcName.indexOf( "outer" ) === 0 ) {
+						return elem[ "inner" + name ];
+					}
+
 					// As of 5/8/2012 this will yield incorrect results for Mobile Safari, but there
```

Comment:
Interestingly enough, I see the bug in iOS 6 in the iPhone 4S simulator but not in the iPhone 5 one...

In any case, I'd assume we don't need to care about it anymore.